source/_docs/environment-specific-config-d8.md

```diff
@@ -0,0 +1,150 @@
+  		case 'live':
+  		case 'test':
+          $config['system.performance']['cache']['page']['use_internal'] = TRUE;
+          $config['system.performance']['css']['preprocess'] = TRUE;
+          $config['system.performance']['css']['gzip'] = TRUE;
+          $config['system.performance']['js']['preprocess'] = TRUE;
+          $config['system.performance']['js']['gzip'] = TRUE;
+          $config['system.performance']['response']['gzip'] = TRUE;
+          $config['views.settings']['ui']['show']['sql_query']['enabled'] = FALSE;
+          $config['views.settings']['ui']['show']['performance_statistics'] = FALSE;
+          $config['system.logging']['error_level'] = 'none';
+		  break;
+  		case 'dev':
+        default :
+          $config['system.performance']['cache']['page']['use_internal'] = FALSE;
+          $config['system.performance']['css']['preprocess'] = FALSE;
+          $config['system.performance']['css']['gzip'] = FALSE;
+          $config['system.performance']['js']['preprocess'] = FALSE;
+          $config['system.performance']['js']['gzip'] = FALSE;
+          $config['system.performance']['response']['gzip'] = FALSE;
+          $config['views.settings']['ui']['show']['sql_query']['enabled'] = TRUE;
+          $config['views.settings']['ui']['show']['performance_statistics'] = TRUE;
+          $config['system.logging']['error_level'] = 'all';
+          # $settings['cache']['bins']['render'] = 'cache.backend.null';
+          # $settings['cache']['bins']['dynamic_page_cache'] = 'cache.backend.null';
+		  break;
+    	}
+  }
+  ```
+
+2. Stage, commit, and push your changes to Pantheon:
+
+ ```
+ git commit -am "Override system performance configurations per env"
+ git push origin master
+ ```
+
+
+<ol start="3"><li>Verify overridden configurations for each config.name on the Dev environment within the Drupal UI using the Configuration Manager core module (<code>/admin/config/development/configuration/single/export</code>) or via <a href="/docs/terminus">Terminus</a>:
+<pre><code>terminus drush 'config-get system.performance --include-overidden'
+terminus drush 'config-get system.logging --include-overidden'
+terminus drush 'config-get views.settings --include-overidden'
+</code></pre>
+<div class="alert alert-info">
+<h4>Note</h4>
+<p>Overridden configurations are not shown within Drupal's admin interface; this behavior is intentional. For details, see <a href="https://www.drupal.org/docs/8/api/configuration-api/configuration-override-system">Configuration override system</a>.</p>
+</div>
+</li></ol>
+
+4. Deploy to Test and verify desired configurations are present. If everything looks good, deploy to Live.
+
+
+Local environments can achieve the same configuration via [Drupal Console](https://drupalconsole.com/). The settings suggested on this page mirror settings applied by [`drupal site:mode dev` and `drupal site:mode prod`](https://hechoendrupal.gitbooks.io/drupal-console/content/en/commands/site-mode.html).
```

Comment:
Should we document the Drupal Console way when Drupal Console is not yet supported on the platform?